asv/environment.py

```diff
@@ -416,7 +416,9 @@ def check_presence(self):
 
         for executable in ['pip', 'python']:
             exe = self.find_executable(executable)
-            if not os.path.isfile(exe):
+            # do not count a system install of pip or python
+            # as evidence of an installed env
+            if not os.path.isfile(exe) or (self._path not in os.path.dirname(exe)):
```

Comment:
This check should be removed: the correct paths must be specified in find_executable.